test/e2e/e2e_test.go

```diff
@@ -70,6 +67,8 @@ func TestMain(m *testing.M) {
 	flag.StringVar(&globalOpts.configurableClusterOptions.Region, "e2e.aws-region", "us-east-1", "AWS region for clusters")
 	flag.Var(&globalOpts.configurableClusterOptions.Zone, "e2e.aws-zones", "Deprecated, use -e2e.availability-zones instead")
 	flag.Var(&globalOpts.configurableClusterOptions.Zone, "e2e.availability-zones", "Availability zones for clusters")
+	flag.StringVar(&globalOpts.configurableClusterOptions.AWSOidcS3BucketName, "e2e.aws-oidc-s3-bucket-name", "", "AWS S3 Bucket Name to setup the OIDC provider in")
+	flag.StringVar(&globalOpts.configurableClusterOptions.AWSKmsKeyAlias, "e2e.aws-kms-key-alias", "", "AWS KMS Key Alias to use when creating encrypted nodepools, when empty the default EBS KMS Key will be used")
```

Comment:
This should probably default with the old value for CI purposes - `alias/hypershift-ci`

Reply:
Similar to the S3 defaulting line of thinking, but in this case I am trying to align with AWS behavior/codebase behavior.

If `nodePool.Spec.Platform.AWS.RootVolume.Encrypted` is set to true, `nodePool.Spec.Platform.AWS.RootVolume.EncryptionKey` is valid if it's an empty value and uses the [default EBS KMS Key](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html#EBSEncryption_key_mgmt)

Reply:
Ok, I'm good with this. I previously missed where it was passed in thru `hack/ci-test-e2e.sh`.